src/js/lib/client/package.js

```diff
@@ -47,6 +59,8 @@ config.build = {
     },
     files: [
         { from: paths.dist.content, to: "." }
+    ,
+        { from: paths.dist.bin, to: '.' },
```

Comment:
What is inside of `paths.dist.bin`? Only project manager now?

Reply:
Yes.